.changeset/funny-glasses-rhyme.md

```diff
@@ -0,0 +1,8 @@
+---
+"@comet/brevo-admin": minor
+"@comet/brevo-api": minor
+---
+
+Add field for `doiTemplateId` to `BrevoConfigPage`
+
+Env var `BREVO_DOUBLE_OPT_IN_TEMPLATE_ID` can be removed
```

Comment:
@RainbowBunchie Does it make sense to add a changeset for every field added to the config? I still have some upcoming PRs that add additional fields (redirectionURL, unsubscribe page, folderId...). Is it possible to update the previous changeset from https://github.com/vivid-planet/comet-brevo-module/pull/112/files instead of adding new ones at least as long as we are working on the next branch?

Reply:
I think it would be best to do this in a feature branch in the future. Then 1 changeset would be enough for the entire feature branch.

I think it would be "okay" as you suggested in this case, but it is less clean. After all, releases are created for each merge, and then the changesets aren't correct anymore I guess.